build_tools/buildkite/pipelines/partial/README.md

```diff
@@ -0,0 +1,5 @@
+# Partial Pipelines
+
+These are Buildkite pipelines intended to be inserted into other pipelines with
+the `buildkite-agent pipeline upload` command. They are not expected to make
+sense on their own.
```

Comment:
bikeshedding on naming: could put a more positive spin on "partial" pipelines than "not expected to make sense on their own"

some other name ideas:
building blocks, components, mixins, plugins, subpipelines, stages, utils/utilities

but could also just tweak the phrasing here like "they are not standalone" (they should still "make sense" and have a clear purpose, but you wouldn't add a top level build that _just_ runs one of these)

Reply:
Renamed directory to "include" and changed the wording